pkg/updater/updater.go

```diff
@@ -1342,6 +1342,10 @@ func alertRow(cols []*statepb.Column, row *statepb.Row, failuresToOpen, passesTo
 		compressedIdx++
 
 		for i := 0; i < len(columnHeader); i++ {
+			if i >= len(col.Extra) {
+				logrus.WithField("started", time.Unix(0, int64(col.GetStarted()*float64(time.Millisecond)))).WithField("additionalColumnHeaders", col.GetExtra()).Trace("Insufficient column header values to record.")
```

Comment:
IIRC logrus lets you do multiple fields at once...

Found example: https://github.com/kubernetes/test-infra/blob/9cd5611d0a632c3f04d2c03cc56d58ebd887b9b7/prow/github/webhooks.go#L74

Reply:
Ah yup, I just usually do it at 3+ fields.

This is pretty long though, so updated!